src/ol_infrastructure/applications/botkube/__main__.py

```diff
@@ -155,7 +155,17 @@
                             "default": {
                                 "name": f"#{slack_channel}",
                                 "bindings": {
-                                    "executors": ["k8s-default-tools"],
+                                    "plugins": {
+                                        "repositories": {
+                                            "botkube": {
+                                                "url": "https://github.com/kubeshop/botkube/releases/download/v1.14.0/plugins-index.yaml"
+                                            },
+                                        },
+                                    },
```

Comment:
I don't think the plugins bit goes under the communications block. I think executors also wants to be its own top-level block at the same level as sources and communications.